### PR TITLE
[docs] add 2 blog posts to readme

### PR DESCRIPTION
**Pull request by matyjas**
### :pencil: Description

Added a new README section for blog posts with two recent posts

### :link: Related Issues

n/a

### CODE REVIEW

**Comment by smyrick** [comment]
Hey @matyjas, thank you for the pull request! I am fine adding this section to the README but what if we linked to the existing `Blogs & Videos` section already on our docs page:

https://expediagroup.github.io/graphql-kotlin/docs/blogs-and-videos

That way we don't have to update the README everytime we post a new blog

**Comment by matyjas** [comment]
@smyrick that makes total sense, didnt see that docs page

File: README.md [hunk 1]
```diff
@@ -39,6 +39,11 @@ If you have a question about something you can not find in our documentation, th
 
 If you would like to contribute to our documentation see the [website](/website) directory for more information.
 
+## 🗞 Blog Posts
+
+* [Introducing GraphQL Kotlin Client](https://medium.com/expedia-group-tech/introducing-graphql-kotlin-client-b32dc3061a6f)
+* [🎉 Announcing graphql-kotlin 2.0!](https://medium.com/expedia-group-tech/graphql-kotlin-2-0-4006ea41f774)
```
Comment (smyrick) [inline README.md:45]:
```suggestion
## 🗞  Blog Posts and Videos

You can view a collection of our public blog posts and videos on our [page in our docs site](https://expediagroup.github.io/graphql-kotlin/docs/blogs-and-videos). These include release announcements, conference talks about the library, and general talks about GraphQL at Expedia Group. 
```

Comment (EXPErdelfavero) [reply README.md:45]: My suggested text:

The [Blogs and videos page](https://expediagroup.github.io/graphql-kotlin/docs/blogs-and-videos) in the GraphQL Kotlin documentation links to blog posts, release announcements, conference talks about the library, and general talks about GraphQL at Expedia Group.

Comment (EXPErdelfavero) [reply README.md:45]: ...and then of course we need to add these two blog post links to that page...

Comment (smyrick) [reply README.md:45]: Those posts are already linked, I updated my suggestion, thanks @EXPErdelfavero !

Comment (matyjas) [reply README.md]: I've reversed the order, so that newest posts are on top, and updated the readme as per suggestions.

**Pull request merged**